site/soc/secrets/passphrases/osh_rabbitmq_erlang_cookie.yaml

```diff
@@ -0,0 +1,11 @@
+---
+schema: deckhand/Passphrase/v1
+metadata:
+  schema: metadata/Document/v1
+  name: osh_rabbitmq_erlang_cookie
+  layeringDefinition:
+    abstract: false
+    layer: site
+  storagePolicy: cleartext
+data: {{ lookup('password', secrets_location + '/osh_rabbitmq_erlang_cookie ' + password_opts) }}
+...
```

Comment:
What's this for? I do not see it used in the code (not mentioning the commit message)